src/index.js

```diff
@@ -125,6 +130,18 @@ class Dropzone extends React.Component {
     }
   }
 
+  handleOnCancel() {
+    const { close } = this.props;
+    const { fileInputEl } = this;
+    setTimeout(() => {
+      const FileList = fileInputEl.files;
+      if (!FileList.length) {
+        document.body.onfocus = null;
+        close();
```

Comment:
Since it isn't a required prop, this will throw an error if it is not specified.